modules/FindMagnum.cmake

```
@@ -410,6 +412,10 @@ endif()
 
 set(_MAGNUM_Trade_DEPENDENCIES )
 set(_MAGNUM_AndroidApplication_DEPENDENCIES GL)
+set(_MAGNUM_EmscriptenApplication_DEPENDENCIES GL)
```

Ummmmmm I guess you wanted to say

```suggestion
set(_MAGNUM_EmscriptenApplication_DEPENDENCIES )
```

instead, right?

---

Yeah 😅